Chaincase.UI/ViewModels/NewPasswordViewModel.cs

```diff
@@ -32,6 +32,7 @@ public void SetPassword(string password)
 
 			Global.UiConfig.HasSeed = true;
 			Global.UiConfig.ToFile();
+			Global.SetDefaultWallet();
```

Comment:
Isn't this redundant to the following? I don't think it causes problems so I'll merge for now

https://github.com/chaincase-app/chaincase/blob/54d5538f512f40b43e675e3993ccdf44cd64d3ac/Chaincase.UI/ViewModels/IndexViewModel.cs#L40-L44